R/outputs.R

```diff
@@ -763,13 +784,22 @@ save_output <- function(filename, dir,
     write_datapack_csv(naomi_output, "pepfar_datapack_indicators_2022.csv")
   }
 
-  dir.create("info")
-  write_navigator_checklist(naomi_output, "info/unaids_navigator_checklist.csv")
 
+  dir.create("info")
   info <- attr(naomi_output, "info")
-  if (length(info) > 0L) {
-    for (p in names(info)) {
-      writeLines(trimws(info[[p]]), file.path("info", p))
+
+  options <- yaml::read_yaml(text = info$options.yml)
+  data <- info$data
+
+  write_navigator_checklist(naomi_output, options, data,
+                            "info/unaids_navigator_checklist.csv")
+
+  info_sub <- info
```

Comment:
@jeffeaton here is where info is saved back out - added this in to stop the additional `data` element from being saved out

Reply:
`data` no longer saved in `info` object in re-factored code.